types/xo/index.d.ts

```diff
@@ -3,6 +3,7 @@
 // Definitions by: Piotr Błażejewicz <https://github.com/peterblazejewicz>
 //                 Chuah Chee Shian (shian15810) <https://github.com/shian15810>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.9
```

Comment:
```suggestion
```
it's by design. Always tested with recent TS support window. The 3.9 is the oldest one anyway. There are no language specific features in this declaration file

Reply:
I think this is for TS publishing/linting tooling only, it has no direct usage.
https://github.com/microsoft/DefinitelyTyped-tools/blob/bf52b08dedb8e7860337a5e1091f42d07d849ffb/packages/dtslint/src/index.ts#L291

Reply:
Hmm, O.K., since the CI passes here, let's just call it a day on this one :) I removed it as it seems like the only trouble it causes is having to put up with the error locally.